### DISCUSSION TITLE
Testsigma Server is not starting due to below error

### DISCUSSION THREAD

**Discussion by sseenivasan89** [testsigmahq/testsigma#236] [Questions & Help (How to)]
i am getting the below error when i start the testsigma server 

**Caused by: com.mysql.cj.exceptions.UnableToConnectException: Public Key Retrieval is not allowed**

**Comment by creek23** [comment]
I got the same problem as well.

specs:
- [TestSigma Server 2.4.1](https://github.com/testsigmahq/testsigma/releases/tag/v2.4.1)
- MySQL 8.0.31

I tried using different MySQL instead (8.0.27 from Wampserver)

**Reply by creek23** [reply]
updated Wampserver, now MySQL is v8.0.31 but Testsigma Server still runs without error.

**Comment by amitsavyasachi** [comment]
@sseenivasan89 @creek23 We have made few changes, can you check now and let us know if you are facing the same issue
